includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -129,6 +129,18 @@ class WC_Gateway_Swedbank_Pay_Checkout extends WC_Payment_Gateway {
 	 */
 	public $checkin_style = '';
 
+	/**
+	 * Send payer info
+	 * @var string
+	 */
+	public $use_payer_info = 'yes';
+
+	/**
+	 * Send cardholder info
+	 * @var string
+	 */
+	public $use_cardholder_info = 'yes';
```

Comment:
Does setting `$use_cardholder_info` to  `'yes'` mean that it is required and must always be sent? It should not be required and should only be sent if it is made available by the WooCommerce installation somehow.